lib/src/utils.dart

```diff
@@ -292,20 +292,27 @@ T maxAll<T extends Comparable>(Iterable<T> iter,
       .reduce((max, element) => compare(element, max) > 0 ? element : max);
 }
 
-/// Like [minBy], but with an asynchronous [orderBy] callback.
+/// Returns the element of [values] for which [orderBy] returns the smallest
+/// value.
+///
+/// Returns the first such value in case of ties.
```

Comment:
Maybe we should write an explicit test case for this utility, to ensure that we cover this?

Reply:
I wrote a test of stable ordering.